app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -486,6 +489,65 @@ class ProfileChooserFragmentTest {
     }
   }
 
+  @Test
+  @RunOn(TestPlatform.ESPRESSO)
+  fun testProfileChooserFragment_clickProfile_checkOpensHomeActivity() {
```

Comment:
in terms of test naming, we follow a pattern
`testSomeFile_condition1_condion2_condition3_expectedResult()`
or
`testSomeBehaviour/Process_condition1_condion2_condition3_expectedResult()`

```suggestion
  fun testProfileChooserFragment_clickProfile_opensHomeActivity() {
```

This applies to all tests in the PR.

Reply:
Done and got it, thanks!